packages/jsii-spec/lib/spec.ts

```diff
@@ -134,13 +134,6 @@ export interface Assembly extends Documentable {
      * @default none
      */
     types?: { [fqn: string]: Type };
-
```

Comment:
No breaking changes to the jsii spec at this time

Reply:
But why though?